Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -1962,6 +2043,155 @@ def cancel_containment_command(client: Client, args: Dict[str, Any]) -> CommandR
     return CommandResults(readable_output=message)
 
 
+"""
+HOST SETS
+"""
+
+
+def delete_host_set_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    host_set_id = args.get('host_set_id')
+
+    if not host_set_id:
+        raise ValueError('Host Set ID is required')
+
+    try:
+        client.delete_host_set_request(host_set_id)
+        message = f'Host set {host_set_id} was deleted successfully'
+    except Exception as e:
+        if '404' in str(e):
+            message = f'Host set id - {host_set_id} Not Found'
+        else:
+            raise ValueError(e)
+
+    return CommandResults(readable_output=message)
+
+
+def create_static_host_set_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    host_set_name = args.get('host_set_name')
+    hosts_ids = argToList(args.get('hosts_ids'))
+
+    if not hosts_ids:
+        raise ValueError('Please provide at least one Host ID.')
+    elif not host_set_name:
+        raise ValueError('Host Set name is required.')
+    data = None
+    try:
+        response = client.create_static_host_set_request(host_set_name, hosts_ids)
+        data = response['data']
```

Comment:
let's make sure data exists in the response, something like:
```suggestion
        if not (data := response.get('data')):
            <do what you need to do here>
```

Reply:
If the request succeeded, there is always a data entry, but I'll put in that check

Reply:
did you mean `if data := response.get('data'):` ?